code/game/objects/structures/shower.dm

```diff
@@ -4,8 +4,12 @@
 #define SHOWER_NORMAL_TEMP 300
 #define SHOWER_BOILING "boiling"
 #define SHOWER_BOILING_TEMP 400
-#define SHOWER_REACTION_MULTIPLIER 0.05
+#define SHOWER_REACTION_MULTIPLIER 0.025
 
+/// Whether the shower can automatically refill with it's reagent.
+#define SHOWER_CAN_REFILL			(1<<0)
+/// Whether players can toggle whether the shower automatically refills with it's reagent.
+#define SHOWER_CAN_TOGGLE_REFILL	(1<<1)
```

Comment:
if you're going to have spurious defines, undefine them at the end of the file please

Reply:
Done